client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1431,6 +1431,8 @@
             },
             templateUrl: 'scripts/superdesk-authoring/views/send-item.html',
             link: function sendItemLink(scope, elem, attrs) {
+                var parentScope = scope.$parent.$parent;
```

Comment:
Yeah, I know, and in first way I do it like this. But (always some but), this directive is also used for sending item on another desk in authoring view. As there is no `sdMonitoringView` directive found by require in authoring screen, it throws error. 

This way was my alternative.